pkg/tcpip/header/icmpv4.go

```diff
@@ -31,6 +31,9 @@ const (
 	// ICMPv4MinimumSize is the minimum size of a valid ICMP packet.
 	ICMPv4MinimumSize = 8
 
+	// Timestamp reply minimum size
```

Comment:
Same comment as above. The comment should be of the form:

// ICMPv4TimeStampMinimumSize is ... .

(Note the end with a ‘.’ as well.)